src/Constants.sol

```diff
@@ -314,6 +314,18 @@ library FraxtalL2 {
     address internal constant STATE_ROOT_ORACLE = 0xeD403d48e2bC946438B5686AA1AD65056Ccf9512;
     address internal constant MERKLE_PRICE_SOURCE = 0x9990eb28E58380122AA3AdF64ea6f874df51114C;
     address internal constant CHAINLINK_MERKLE_PRICE_SOURCE = 0x4AD728706bFe28A3F090b3f1D8Ac3A8515805B41;
+    address internal constant SFRXETH_L2_RATE_ORACLE = 0xEE095b7d9191603126Da584a1179BB403a027c3A;
+    address internal constant SFRAX_L2_RATE_ORACLE = 0x0D7888e8aC9e5D0915148E0F871a3893aF3d2aaF;
+    address internal constant SDAI_L2_RATE_ORACLE = 0xfdE8C36F32Bf32e73A1bdeb4ef3E17709674a838;
+    address internal constant SUSDE_L2_RATE_ORACLE = 0xd295936C8Bb465ADd1eC756a51698127CB4F4910;
+    address internal constant SFRXETH_L2_RATE_ORACLE_OLD = 0x8865435777730eAAbAAF2d1F55F115a87AbCf91A;
```

Comment:
This should be fine assuming there's no real need to actively check the old oracle as a second update would overrwrite the old oracle and require looking at the diff history.

Reply:
Good catch there is no need to update/point to `0x0D7888e8aC9e5D0915148E0F871a3893aF3d2aaF`

Reply:
I'm confused here as I was referring to the use of `_OLD` variables, although if there's a fix from this comment that works too.

Reply:
@pegahcarter Yeah so the reason we have the old Oracles is becuase they are still being used by the curve pools on fraxtal and are declared as immutable. 

As a consequence, we will still support them going forward. Unless we want to do a liquidity migration on these pools. But none of the changes from the audit impact the manner in which these oracles are utilized by the pools.